playbulb-candle/playbulbCandle.js

```diff
@@ -114,6 +116,20 @@
         this._isEffectSet = true;
       });
     }
+    startBlowNotifications(listener) {
+      let characteristic = this._characteristics.get(CANDLE_BLOW_NOTIFICATIONS_UUID);
+      return characteristic.startNotifications()
+      .then(_ => {
+        characteristic.addEventListener('characteristicvaluechanged', listener);
+      });
+    }
+    stopBlowNotifications(listener) {
+      let characteristic = this._characteristics.get(CANDLE_BLOW_NOTIFICATIONS_UUID);
+      return characteristic.stopNotifications()
+      .then(_ => {
+        characteristic.removeEventListener('characteristicvaluechanged', listener);
```

Comment:
The only reason to remove the listener after was that I thought `stopNotifications()` could reject.
Removing it after would simply prevent a characteristic still notifying and an listener still there. Then, next time, I'd call `startNotifications()`, there would be two listeners. Does that make sense?